src/verification-method/model/verification-method.model.ts

```diff
@@ -0,0 +1,60 @@
+import { chainIdOf } from "@lto-network/lto-crypto";
+
+export interface DIDVerificationMethod {
```

Comment:
Move this to the same file as the DIDDocument interface

Reply:
done